sycl/include/CL/sycl/INTEL/esimd/esimd_math.hpp

```diff
@@ -1838,7 +1838,7 @@ template <typename T0, typename T1, int SZ> struct esimd_apply_reduced_min {
 
 template <typename T0, typename T1, int SZ,
           template <typename RT, typename T, int N> class OpType>
-T1 esimd_reduce_single(simd<T1, SZ> v) {
+T0 esimd_reduce_single(simd<T1, SZ> v) {
```

Comment:
This change does not seem to be related to raw_send.

Reply:
Right, as you can see from the change list, this patch includes other local changes including ESIMD tests update and a reduction test fix.